src/_posts/security/procedures/2000-01-01-external-testing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Application external testing
+nav: External testing
+modified_at: 2022-03-15 00:00:00
+tags: security pentest load-test procedures
+---
+
+## Testing my application that is running on Scalingo
+
+### Can I run a load test on my application that is running on Scalingo?
+
+You can run a load test / smoke test / endurance test on your application under the following conditions :
```

Comment:
There is no space before semi-colon, exclamation or interrogation point in English. You have this error at different places of this PR

```suggestion
You can run a load test / smoke test / endurance test on your application under the following conditions:
```

Reply:
Classic me!